modules/contact-form/grunion-contact-form.php

```diff
@@ -1028,9 +1028,32 @@ static function parse( $attributes, $content ) {
 		return $r;
 	}
 
+	/**
+	 * Returns a success message to be returned if the form is sent via AJAX.
+	 *
+	 * @param Integer $feedback_id
+	 * @param Grunion_Contact_Form $form
+	 *
+	 * @return String $message
+	 */
 	static function success_message( $feedback_id, $form ) {
-		$r_success_message = '';
+		return wp_kses(
+			"<blockquote>\n"
```

Comment:
Also, I see us using `\n` (and that was already there) -- isn't it better practice to use `PHP_EOL` for wider compatibility?